boot/bootdata/txtsetup.sif

```diff
@@ -90,7 +90,7 @@ FreeSysPartDiskSpace=450
 acpi.sys     = 1,,,,,,,4,1,,,1,4
 nmidebug.sys = 1,,,,,,x,4,,,,1,4
 sacdrv.sys   = 1,,,,,,x,4,,,,1,4
-uniata.sys   = 1,,,,,,x,4,,,,1,4
+atapi.sys   = 1,,,,,,x,4,,,,1,4
```

Comment:
```suggestion
atapi.sys    = 1,,,,,,x,4,,,,1,4
```

Reply:
This aligned list is a good example by the way why sticking with 8.3 names (whereever possible) is a nice thing to have, @learn-more